### PR TITLE
Improvement: DO-2837 dara to use dara UI markdown component

### PR DESCRIPTION
**Pull request by patricia-causalens**
## Motivation and Context

To use dara-ui Markdown component, as to avoid duplicates across the product

## Implementation Description

Updated to use dara-ui Markdown component

## Any new dependencies Introduced

None

## How Has This Been Tested?

Locally

## PR Checklist:

- [x] I have implemented all requirements? (see JIRA, project documentation).
- [x] I am not affecting someone else's work, If I am, they are included as a reviewer.
- [ ] I have added relevant tests (unit, integration or regression).
- [ ] I have added comments to all the bits that are hard to follow.
- [ ] I have added/updated Documentation.
- [x] I have updated the appropriate changelog with a line for my changes.

### CODE REVIEW

File: packages/dara-components/package.json [hunk 1]
```diff
@@ -72,17 +72,15 @@
     "prism-react-renderer": "^1.2.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-markdown": "^9.0.1",
     "react-plotly.js": "~2.6.0",
     "react-virtualized-auto-sizer": "1.0.6",
     "recoil": "^0.7.7",
-    "rehype-raw": "^6.1.1",
-    "remark-gfm": "^4.0.0"
+    "rehype-raw": "^6.1.1"
```
Comment (krzysztof-causalens) [inline packages/dara-components/package.json:78]: I don't think we need the plugin as a direct dependency anymore?

**Review by krzysztof-causalens — APPROVED** [review]

File: packages/dara-components/package.json (hunk 1 above)
Comment (patricia-causalens) [reply packages/dara-components/package.json:78]: We still need rehypeRaw plugin as that is not added on dara-ui side, but the gfm plugin was removed

**Review by tamara-causalens — APPROVED** [review]
lovely!

File: packages/dara-components/package.json (hunk 1 above)
Comment (krzysztof-causalens) [reply packages/dara-components/package.json:78]: Right, makes sense

**Pull request merged**